### ISSUE TITLE
accession numbers unmapped

### ISSUE THREAD

**Issue by fujch7** [DerrickWood/kraken2#713]
Hi, developer:

I downloaded NT database and taxonomy files manually and run kraken2-build command. The log said:

   **lookup_accession_numbers: 60132/84015636 accession numbers remain unmapped, see unmapped.txt in DB directory**

In step3 Building database files, will kraken2 discard the unmapped sequences?

**Comment by palatinate** [comment]
Could you share some of the fasta headers?

**Comment by ch4rr0** [comment]
Yes unmapped accessions will be ignored. The seqid2taxid.map file will contain the senquence names that kraken will use for building the database.
